compiler/src/Language/Bond/Codegen/Java/Class_java.hs

```diff
@@ -373,6 +373,41 @@ public #{declName}(com.microsoft.bond.StructBondType<#{declName}<#{paramList}>>
             else mempty
 
 
+-- builds text for Object.equals(Object) override
+object_equals :: String -> [Field] -> Text
+object_equals declName fields = [lt|@Override
+    public boolean equals(Object o) {
+        if (o == null) return false;
+        if (!(o instanceof #{declName})) return false;
+        final #{declName} other = (#{declName}) o;
+        #{newlineSep 2 compareField fields}
```

Comment:
>#{newlineSep 2 compareField fields} [](start = 8, length = 35)

If there are no fields, this will produce a blank line. Can you move this to the previous or next line?